utils/extra_config.py

```diff
@@ -3,6 +3,28 @@
 import folder_paths
 import logging
 
+from .json_util import merge_json_recursive
+
+default_server_config = {
+    'internal': {
```

Comment:
I am not sure if it is the best place to put the config default, does anybody has a better suggestion ?